### PR TITLE
Update submodules to pick up major changes from nnf-sos

### PR DESCRIPTION
**Pull request by bdevcich**

### CODE REVIEW

**Review by ajfloeder — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -15,10 +15,10 @@ require (
 )
 
 require (
-	github.com/NearNodeFlash/lustre-fs-operator v0.0.1-0.20231031201943-531116c1194e
-	github.com/NearNodeFlash/nnf-dm v0.0.1-0.20231031212032-66e6f08012fc
+	github.com/NearNodeFlash/lustre-fs-operator v0.0.1-0.20231115220321-eb84599f9630
+	github.com/NearNodeFlash/nnf-dm v0.0.6
```
Comment (roehrich-hpe) [inline go.mod:19]: You wanted to pin to a release?

File: go.mod [hunk 2]
```diff
@@ -3,7 +3,7 @@ module github.com/NearNodeFlash/nnf-integration-test
 go 1.19
 
 require (
```
Comment (roehrich-hpe) [inline go.mod:5]: I wonder if it'll be happy if you add a collection of 'replace' lines after line 4:

```
replace github.com/DataWorkflowServices/dws => ../dws
replace github.com/NearNodeFlash/lustre-fs-operator => ../lustre-fs-operator
...
```

Comment (roehrich-hpe) [reply go.mod:5]: But then it can't be used outside of an nnf-deploy workarea, so scratch that.

**Review by roehrich-hpe — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (bdevcich) [reply go.mod]: No, not sure what happened there, but it's fixed now.

**Review by roehrich-hpe — APPROVED** [review]

**Pull request merged**